bin/fusor-installer

```diff
@@ -1,90 +1,226 @@
 #!/bin/bash
 
-if [ "$(id -u)" != "0" ]; then
-   echo "This script must be run as root"
-   exit 1
-fi
-
-h=\\-h\\b
-fusor-setup $@
-if [ $? != 0 ]; then
-  echo "fusor-setup did not run properly."
-  echo "Please correct the problem and try again."
-  exit 2
-elif [[ $@ == *"--help"* ]] || [[ $@ =~ $h ]] || [[ $@ == *"--full-help"* ]]; then
-  exit 0
-fi
-
-#kafo nukes the file permissions whatever we make them.
-if [[ $@ != *"--devel_env"* ]]; then
-  chmod 660 /etc/fusor-installer/fusor-installer.answers.yaml
-  chown root:foreman /etc/fusor-installer/fusor-installer.answers.yaml
-fi
-
-echo "Finished running fusor-setup."
-echo "Now running foreman-installer."
-
-foreman-installer --scenario fusor
-
-if [ $? != 0 ]; then
-  echo "foreman-installer did not run properly."
-  echo "Please correct the problem and try again."
-  exit 3
-fi
-
-echo "Finished running foreman-installer."
-echo "Now running rake fusor:setup."
-
-if [[ $@ == *"--devel_env"* ]]; then
-  echo "Setting up file permission for development environment."
-  setfacl -m u:vagrant:r /var/log/messages
-  setfacl -m u:vagrant:rx /var/log/foreman-proxy
-else
-  echo "Setting up file permission for production environment."
-  setfacl -m u:foreman:r /var/log/messages
-fi
-
-if [[ $@ == *"--devel_env"* ]]; then
-  while [[ $# > 0 ]]; do
-    i="$1"
-    case $i in
-      --deployment_dir=*)
-      deployment_dir="${i#*=}"
-      shift
-      ;;
-      --deployment_dir)
-      deployment_dir="$2"
+fusor_install() {
+  if [ "$(id -u)" != "0" ]; then
+     echo "This script must be run as root"
+     exit 1
+  fi
+
+  interactive=true
```

Comment:
Where do we unset ```interactive```? We set it to true here and check it 3 times below but never do we set it to false that I can see.